io/www/src/fragment/paths.js

```diff
@@ -20,8 +20,7 @@ function odinId(id) {
  * @returns full fetchable path to the fragment references
  */
 function odinReferences(id, allHydrated = false) {
-    /* istanbul ignore next */
-    return `${odinId(id)}/variations/master/references${allHydrated ? '?references=all-hydrated' : ''}`;
+    return `${odinId(id)}${allHydrated ? '?references=all-hydrated' : ''}`;
```

Comment:
for a start, just checking: have you tested this (removal of master variation specification) works with actual usages of it: 
- placeholders/index request,
- collection request (i guess yes :))

if yes, i think it's better now, as we may never use CF variation mechanism anyhow

Reply:
I didn't specifically test removing master.  But it works with the collection request. Happy to discuss the variation use case if you think we might use it at all.

Reply:
i'm good in the scope of that request